data-protocols/ids/ids-api-multipart-dispatcher-v1/src/test/java/org/eclipse/dataspaceconnector/ids/api/multipart/dispatcher/sender/IdsMultipartSenderTest.java

```diff
@@ -47,9 +45,9 @@ void should_fail_if_token_retrieval_fails() {
 
         var objectMapper = IdsTypeManagerUtil.getIdsObjectMapper(new TypeManager());
 
-        var sender = new TestIdsMultipartSender("any", mock(OkHttpClient.class), objectMapper, mock(Monitor.class), identityService, mock(IdsTransformerRegistry.class));
+        var sender = new IdsMultipartSender(mock(Monitor.class), mock(OkHttpClient.class), identityService, objectMapper);
 
-        var result = sender.send(new TestRemoteMessage(), () -> "any");
+        var result = sender.send(new TestRemoteMessage(), new TestSenderDelegate());
```

Comment:
Could you use a `mock` instead of the `TestSenderDelegate`? Seems it does nothing, and a mock would make the intent clearer IMO.

Reply:
done